docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -76,98 +116,108 @@ all the YAML files deployed in this sample to point at that namespace.
      - kind: ServiceAccount
        name: events-sa
        namespace: default
+   EOF
    ```
 
-   If you want to re-use an existing Service Account with the appropriate permissions,
-   you need to modify the `serviceaccount.yaml`.
+   {{< /tab >}}
 
-2. Enter the following command to create the service account from `serviceaccount.yaml`:
+   {{% tab name="By filename" %}} Use following command to create the service
+   account from the `serviceaccount.yaml` file:
 
    ```shell
    kubectl apply --filename serviceaccount.yaml
    ```
 
-### Create Event Source for Kubernetes Events
+   If you want to re-use an existing Service Account with the appropriate
+   permissions, you need to modify the `serviceaccount.yaml`.
 
-1. In order to receive events, you have to create a concrete Event Source for a
-   specific namespace. Create a file named `k8s-events.yaml` and copy the code
-   block below into it.
+   {{< /tab >}} {{< /tabs >}}
 
-   ```yaml
-   apiVersion: sources.knative.dev/v1alpha1
-   kind: ApiServerSource
-   metadata:
-     name: testevents
-     namespace: default
-   spec:
-     serviceAccountName: events-sa
-     mode: Resource
-     resources:
-       - apiVersion: v1
-         kind: Event
-     sink:
-       ref:
-         apiVersion: eventing.knative.dev/v1alpha1
-         kind: Broker
-         name: default
+2) Enter the following command to create the service account from
```

Comment:
Not sure why this is a step 2 given the above tabs.

Reply:
this was the old formatting, maybe I missed some   👀